README.md

```diff
@@ -54,248 +99,292 @@ $(SRCROOT)/Carthage/Build/iOS/CryptoSwift.framework
+|---|---|
+| **userId** <br> **String**	| Unique identifier of the user stored in OST Platform |
+| **delegate** <br> **OstWorkflowDelegate**	| An object that implements the callback function available in `OstWorkflowDelegate` protocol. These callback functions are needed for communication between app and wallet SDK. Implement `flowComplete` and `flowInterrupt` callback functions to get the workflow status. Details about other callback function can be found in [OstWorkflowDelegate protocol reference](/platform/docs/sdk/references/wallet_sdk/iOS/latest/protocols/). |
 
-A user that has stored their mnemonic phrase can enter it into an appropriate user interface on a new mobile device and authorize that device to be able to control their Brand Tokens.<br/><br/>
-**Parameters**<br/>
-&nbsp;_userId: OST Platform user id provided by application server_<br/>
-&nbsp;_mnemonics: Array of mnemonics_<br/>
-&nbsp;_delegate: Callback implementation object for application communication_<br/>
 
-```Swift
-OstWalletSdk.authorizeCurrentDeviceWithMnemonics(
+
+<br>
+
+### 7. executeTransaction
+Workflow should be used to do the `user-to-company` and `user-to-user` transactions.
+
+```
+OstWalletSdk.executeTransaction(
     userId: String,
-    mnemonics: [String],
+    tokenHolderAddresses: [String],
+    amounts: [String],
+    transactionType: OstExecuteTransactionType,
+    meta: [String: String],
+    options: [String: Any],
     delegate: OstWorkflowDelegate
 )
 ```
 
-### Generate a QR Code
+| Parameter | Description |
+|---|---|
+| **userId** <br> **String**	| Unique identifier of the user stored in OST Platform|
+| **tokenHolderAddresses** <br> **[String]**	|  **TokenHolder**  addresses of beneficiary users.  |
+| **amounts** <br> **[String]**	| Array of Amount to be transferred in atto.  |
+| **transactionType** <br> **OstExecuteTransactionType**	| Transaction type can take one of the two values: <br> 1. `DirectTransfer`:  In this type of transaction, the amount of brand token will be transferred directly to the receiver user. <br> 2. `Pay`: In this type of transaction the amount of fiat passed will first be converted into brand token and after this conversion the transfer will happen in converted brand token amount.|
+| **meta** <br> **[String: String]**	| Dictionary object having extra information that a developer can pass about the transfer. This dictionary object can have 3 properties. <br><br>Example meta:  <br>[<br>&nbsp; &nbsp;"name":"Thanks for like", <br>&nbsp; &nbsp;"type": "user_to_user", <br>&nbsp; &nbsp;  "details": "like"<br>] |
+| **options** <br> **[String: Any]**	| Optional settings parameters. You can set following values: <br> 1. `currency_code`: Currency code for the pay currency. <br> 2. `wait_for_finalization`: If set `false` then SDK will stop polling for transaction status. By default the SDK will do polling to check the transaction status. <br> Example: `{"currency_code": "USD", "wait_for_finalization": false}`|
```

Comment:
We introduced `wait_for_finalization` for our own internal use and are now learning the disadvantages of it.
I think we should not mention it in the doc.

The issue we are facing:
If wait_for_finalization is set to false, the sdk (in current form) has to assume that all transactions are successful. Due to this we have observed that nonce of session keys do go out of sync.

The only way to know that nonce are out sync is to wait for finalization.